build.sbt

```
@@ -14,7 +14,7 @@ val circeVersion: Option[(Long, Long)] => String = {
 val sttpVersion = "3.3.15"
 val Specs2Version = "4.2.0"
 val artifactory = "https://cognite.jfrog.io/cognite/"
-val cogniteSdkVersion = "1.5.14"
+val cogniteSdkVersion = "1.5.16-SNAPOSHOT"
```

```suggestion
val cogniteSdkVersion = "1.5.16"
```
To be updated once new maven release is done